builder/dockerfile/parser/testfiles/escapes/Dockerfile

```diff
@@ -6,9 +6,7 @@ RUN apt-get \update && \
 ADD \conf\\" /.znc
 
 RUN foo \
-
 bar \
-
```

Comment:
I see you test for it in the integration tests, but let's add one to the unit tests too since we should test the parser in isolation of the builder.